src/pybind/mgr/dashboard/frontend/src/app/shared/enum/multisite-wizard-steps.enum.ts

```diff
@@ -0,0 +1,19 @@
+export enum StepTitles {
+  CreateRealmAndZonegroup = 'Create Realm & Zonegroup',
+  CreateZone = 'Create Zone',
+  SelectCluster = 'Select Cluster',
+  Review = 'Review'
+}
+
+export const STEP_TITLES_MULTI_CLUSTER_CONFIGURED = [
+  StepTitles.CreateRealmAndZonegroup,
+  StepTitles.CreateZone,
+  StepTitles.SelectCluster,
+  StepTitles.Review
+];
+
+export const STEP_TITLES_SINGLE_CLUSTER = [
+  StepTitles.CreateRealmAndZonegroup,
+  StepTitles.CreateZone,
+  StepTitles.Review
+];
```

Comment:
I don't understand the value/sense of this. It's reverting how OOP/encapsulation works. Instead of keeping each implementation's files together and separated from the abstract/base clase, it merges some stuff of each implementation here and pretends it's "abstract" by placing it in a shared dir...